### PR TITLE
Cryo Rework: Auto-eject removed, Chemical x10 removed, Cryox/Clonex removed

### PR DESCRIPTION
**Pull request by Iamgoofball**
>you could multiply all chems by 10
>cryo healed everything if mixed right
>auto-eject allows me to be stuffed in/heal/leave no effort

ahahahahahaha jfc

cryox/clonex removed
beaker starts with 15char/15bicard for now

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
I could swear you've tried removing cloner eject several times already...

**Comment by DoctoreMad** [comment]
Quit while you're ahead.

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -173,24 +171,18 @@
...
 			mob_occupant.Sleeping((mob_occupant.bodytemperature / sleep_factor) * 2000)
 			mob_occupant.Unconscious((mob_occupant.bodytemperature / unconscious_factor) * 2000)
 		if(beaker)
-			if(reagent_transfer == 0) // Magically transfer reagents. Because cryo magic.
-				beaker.reagents.trans_to(occupant, 1, 10 * efficiency) // Transfer reagents, multiplied because cryo magic.
-				beaker.reagents.reaction(occupant, VAPOR)
-				air1.gases[/datum/gas/oxygen][MOLES] -= 2 / efficiency //Let's use gas for this
-			if(++reagent_transfer >= 10 * efficiency) // Throttle reagent transfer (higher efficiency will transfer the same amount but consume less from the beaker).
-				reagent_transfer = 0
+			beaker.reagents.trans_to(occupant, 1)
```
Comment (ChangelingRain) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: this should probably still magically create reagents, just in much lower amounts.

Comment (ChangelingRain) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: like, instead of 1:10 something like 2:5

**Comment by Alecw155** [comment]
if you make 90 PRs in a day, surely ONE will be merged, right?

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: i'm iffy about that but eh, i guess we'll give it a try

**Comment by swindly** [comment]
The reactions for synthmeat and colorful reagent use cryoxadone.

**Comment by Iamgoofball** [comment]
they shouldn't be using it, wtf

**Comment by Iamgoofball** [comment]
i'll adjust that thanks for the heads up

**Comment by ChangelingRain** [comment]
this is why key-based reagents suck ass by the way, removing a reagent is a mess

**Comment by calibraptor** [comment]
No

**Comment by Iamgoofball** [comment]
Yeah the reagent system should be using types

**Comment by nicbn** [comment]
Hahahaha no

**Comment by nicbn** [comment]
Ok in case you are not joking

* Do not remove cryoxadone
* Make cryoxadone regenerate you only when at cold temperatures, the colder the more regeneration
* Buff low pressure damage to punish those who spacewalk with cryoxadone

**Comment by Iamgoofball** [comment]
That's missing the point so hard

**Comment by TribeOfBeavers** [comment]
If you're removing clonex/cryox we'll need some other way to heal cellular damage (or is that being removed too?)

**Comment by Iamgoofball** [comment]
Upgrade your cloner

**Comment by TribeOfBeavers** [comment]
Slimes do cellular damage too...

**Comment by swindly** [comment]
Rezadone treats cellular damage.

**Comment by feemjmeem** [comment]
i don't understand why you think these are good ideas

**Comment by Iamgoofball** [comment]
@feemjmeem
I don't understand why you haven't read the OP

**Comment by TribeOfBeavers** [comment]
It requires carpotoxin, which makes it unobtainable most rounds. It can only be made through botany, getting the carp event, or emagging a chem dispenser.

I don't necessarily mind the idea that getting cloned results in a likely permanent debuff, but I think the cellular damage values should be rebalanced a bit if this is the case.

**Comment by Grazyn** [comment]
"upgrade your cloner" is still the correct answer, if you have cellular damage just hop in a brand new body.

**Comment by Blute1** [comment]
epic

**Comment by Buggy123** [comment]
Alternative solution: replace cryo with [showers](https://github.com/tgstation/tgstation/pull/15679). No chem dupe, ezpz.

**Comment by Bawhoppen** [comment]
I think we'd have better luck if we just removed cryo upgrades. That would have a chance at merging unlike this.

**Comment by Iamgoofball** [comment]
@Bawhoppen i'm already doing battle with R&D in 3 different PRs, a 4th is a bad idea

**Comment by Bawhoppen** [comment]
Yea but the issue is that this is too extreme and just making non-R&D enemies aswell, which will hurt your overall chances of any of this stuff being accepted.

**Comment by Iamgoofball** [comment]
the only person that matters in regard to this is kor

it's going to suck not having crutches, but you'll adapt

**Comment by Bawhoppen** [comment]
Kor takes other people's opinions into account though aswell.
And personally I see this as too far of an overreaction.

But I truly applaud you for waging war on the creep.

**Comment by MMMiracles** [comment]
This actually isn't that terrible of a change when you think about it for a minute.

Rezadone suddenly becomes much more valuable and might actually get made now that cryo can't immediately fix all your issues. You can still get perfect cloning from an upgraded cloner, but now doing stupid shit and getting yourself killed early-mid into the round might actually come with repercussions past the inconvenience of the cloning wait time. 

It'd be neat to see this at least testmerged for a bit to see the general response past the initial backlash from the change.

**Comment by Iamgoofball** [comment]
There's a lot of stuff that'll need testmerged at once

**Comment by lordpidey** [comment]
An alternative idea...

Make Cryo tubes not insert reagents, but rather use the TOUCH application method.

And then make Cryoxodone heal upon TOUCH (Regardless of amount) when the patient is also cold.

Further, make Cyroxodone moderately toxic to have in your bloodstream.

This will get rid of Cryox spacewalk shenanigans.

**Comment by PKPenguin321** [comment]
What's wrong with clonex, just curious
It's incredibly situational and no longer a cure all
Also I think this removes every reasonable cure for clone damage

**Comment by Grazyn** [comment]
Clone damage is normally only caused by cloning or slimes. Upgrading the cloner removes the damage from cloning. Being attacked by a slime is extremely rare outside of science, but science can also make carps and the chemists can use carpmeat for rezadone. Also, if you get heavily damaged by a slime, you can just kill yourself and get a new perfect body from the upgraded cloner. If the cloner isn't upgraded, well maybe you should've done some research instead of playing with slimes.

**Comment by Arianya** [comment]
Killing yourself and recloning is not intuitive or a good solution, not to mention a huge time sink

**Comment by Iamgoofball** [comment]
Rezadone should be more easily available

**Comment by Iamgoofball** [comment]
Additionally a soy bean mutation has carpotoxin, fyi

**Comment by Grazyn** [comment]
@Arianya Have you seen classic sci-fi movie "The Sixth Day" with Schwarzenegger? The funniest thing was henchmen killing themselves whenever they got crippled so that they could get cloned in a new body

**Comment by Arianya** [comment]
@Grazyn While sci-fi references are cool, gameplay effect does come into it. Recloning exposes the person being cloned to theft (which they can't know the culprit of because knowledge gained while dead) as well as being fucked with in one of innumerable ways.

Not to mention this makes heavy reliance on R&D upgrading something, which just creates more emphasis on attacking/destroying R&D ASAP, something I was under the impression that goof was trying to move away from.

**Review by imtakingabreakdontatme — CHANGES_REQUESTED** [review]
"Upgrade your cloner" can't be the answer we can't have an unhealable damage type you can only fix by finishing RnD and then killing yourself

**Comment by Iamgoofball** [comment]
I'll make rezadone easier to get 

Then again is cloneloss really a critical mechanic, it exists just to give purpose to cryo

**Comment by sawrge** [comment]
I think it would be interesting if cloning had permanent or lasting effects to make death a little more meaningful although perhaps in a further/separate PR rather than leaving it to the terrible R&D upgrades the genetics minigame could be altered/reworked into minimizing clone damages somehow

**Comment by Buggy123** [comment]
>you can very slowly clone chemicals with cryo
> requires a extremely convoluted process involving 2 separate departments and a supply of monkeys/lizards
>lets completely remove cryo, cryo chems, and a entire damage type from the game
K.

**Comment by TribeOfBeavers** [comment]
I thinm this PR is more about trying to fix medbay gameplay in general than specifically trying to fix the chem cloning thing.

**Comment by KorPhaeron** [comment]
@Iamgoofball I think cloneloss only exists to stop speedcloning but now that we make people start without limbs and organs it might not be as critical anymore

**Comment by Buggy123** [comment]
Then why does the OP say this is entirely about removing chem cloning? I look at the OP for a summary of the PR, and 9/10 times judge it by that summary.

**Comment by jughu** [comment]
Dont make rezadone easier goof let darn botanists/chemists cooperatw for once

**Comment by PKPenguin321** [comment]
@Iamgoofball why is rezadone, a chem that is easier to apply and essentially an upgrade from our current form of clonex, preferable to just keeping clonex?

**Comment by PKPenguin321** [comment]
For the record:
`Clonex: Quickly heals clone damage, but only when in a cold environment.`
`Rezadone: Instantly heals all cellular damage, while healing brute and burn damage over time. If overdosed deals toxin damage. Very good for speedcloning.`
Why do you keep making medbay so hugboxed?

**Comment by TribeOfBeavers** [comment]
@Buggy123

The quote in the OP is 3 different points.


1.Chem cloning
2.Cryo is a cure-all
3.Auto eject allows cryo to be used with almost no effort on the doctor's part

**Comment by jughu** [comment]
Because rezadone is harder to aquire and requires cooperation which never happens because.
Botanists want to make death plants
Chemists wants to make death chems

**Comment by PKPenguin321** [comment]
>Because rezadone is harder to aquire

If you read his posts, you'd see that he intends to make it easier to acquire (presumably roundstart easy) to make up for removing clonex, for some unknown reason (because he has no idea what he's doing).

**Comment by Iamgoofball** [comment]
taking another approach

**Pull request closed without merging**